awx/main/scheduler/task_manager.py

```diff
@@ -620,4 +658,14 @@ def schedule(self):
                 logger.debug("Starting Scheduler")
                 with task_manager_bulk_reschedule():
                     self._schedule()
+                if not settings.IS_TESTING(sys.argv):
```

Comment:
not sure I understand this one setting

Reply:
ok I get it now that I found...don't collect these metrics if tests are running https://github.com/ansible/awx/blob/78660ad0a22798ace9210b37ba5be9429603e49d/awx/settings/defaults.py#L16-L31